src/OS/QMidi_ALSA.cpp

```diff
@@ -9,6 +9,8 @@
 #include <alsa/seq.h>
 #include <alsa/seq_midi_event.h>
 
+/* -----------------------------[ QMidiOut ]------------------------------ */
```

Comment:
Please use the `// #pragma mark - QMidiOut` style, this is compatible with more editors.